olca-proto-io/src/main/java/org/openlca/proto/io/output/ProcessWriter.java

```diff
@@ -61,6 +55,50 @@ public ProtoProcess write(Process process) {
     return proto.build();
   }
 
+	private ProtoProcessDocumentation.Builder mapDocOf(ProcessDocumentation d) {
+		var proto = ProtoProcessDocumentation.newBuilder();
+		if (d == null)
+			return proto;
+
+		proto.setTimeDescription(d.time);
+		proto.setTechnologyDescription(d.technology);
+		proto.setDataCollectionDescription(d.dataCollectionPeriod);
+		proto.setCompletenessDescription(d.completeness);
+		proto.setDataSelectionDescription(d.dataSelection);
+		proto.setReviewDetails(d.reviewDetails);
+		proto.setDataTreatmentDescription(d.dataTreatment);
+		proto.setInventoryMethodDescription(d.inventoryMethod);
+		proto.setModelingConstantsDescription(d.modelingConstants);
+		proto.setSamplingDescription(d.sampling);
+		proto.setRestrictionsDescription(d.restrictions);
+		proto.setIsCopyrightProtected(d.copyright);
+		proto.setIntendedApplication(d.intendedApplication);
+		proto.setProjectDescription(d.project);
+		proto.setGeographyDescription(d.geography);
+		proto.setCreationDate(date(d.creationDate));
+		proto.setValidFrom(date(d.validFrom));
+		proto.setValidUntil(date(d.validUntil));
+
+		proto.setReviewer(Refs.refOf(d.reviewer));
+		proto.setDataDocumentor(Refs.refOf(d.dataDocumentor));
```

Comment:
please use the `config.dep` method here as for the other model references, it does not only handle `null` values correctly but also pushes the references to a dependency handler if it is registered, this is very important when an object graph and its dependencies should be converted, e.g. when exported to a zip file